code/game/objects/items.dm

```diff
@@ -34,6 +36,8 @@ GLOBAL_DATUM_INIT(fire_overlay, /mutable_appearance, mutable_appearance('icons/e
 	var/greyscale_config_inhand_left
 	///The config type to use for greyscaled right inhand sprites. Both this and greyscale_colors must be assigned to work.
 	var/greyscale_config_inhand_right
+	///The config type to use for greyscaled belt overlays. Both this and greyscale_colors must be assigned to work.
+	var/greyscale_config_belt
```

Comment:
Is there really no better way to do this than adding a new variable for every article of clothing? Or is this just a side effect of inventory code being bad?

Reply:
This is a side effect of inventory code being bad, mob overlays for items are all stored in massive icon files like belt_overlays.dmi and lefthand_overlays.dmi which results in item code being unable to support having all the sprites for a single item in one dmi file which we could generate in one GAGS config.

The good part of this though is that from our perspective, GAGS items can have all their sprites in a single dmi and you just need multiple configs to split it up for the various parts of item code. This means we can move stuff out of those titan dmi files and into their own. When eventually we refactor item code to take apart those files all GAGS items will have to do is merge together those configs into one.